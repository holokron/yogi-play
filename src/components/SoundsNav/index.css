.sounds-nav {
  font-size: 13px !important;
}
