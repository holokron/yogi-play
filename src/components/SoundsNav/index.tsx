import { ReactElement } from "react";
import "./index.css";
import SoundNavItem from "../../components/SoundNavItem";
import Tag from "../../types/Tag";
import useTags from "../../hooks/useTags";
import { Nav } from "reactstrap";

export default function SoundsNav(): ReactElement {
  const { currentTag, tags, changeTag } = useTags();

  return (
    <Nav pills className="sounds-nav">
      {tags.map((tag: Tag) => (
        <SoundNavItem
          key={tag.slug}
          onClick={() => changeTag(tag.slug)}
          isActive={!!currentTag && currentTag.slug === tag.slug}
        >
          {tag.name}
        </SoundNavItem>
      ))}
    </Nav>
  );
}
