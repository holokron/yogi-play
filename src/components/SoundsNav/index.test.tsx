import * as React from "react";
import * as renderer from "react-test-renderer";
import { MemoryRouter } from "react-router-dom";
import SoundsNav from ".";
import configureStore from "../../store";
import { Provider } from "react-redux";

const store = configureStore();

describe("@components/SoundsNav", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <MemoryRouter>
            <SoundsNav />
          </MemoryRouter>
        </Provider>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
