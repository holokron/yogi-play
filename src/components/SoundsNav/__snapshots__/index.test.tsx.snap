// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@components/SoundsNav renders correctly 1`] = `
<ul
  className="sounds-nav nav nav-pills"
/>
`;
